src/v2/Apps/BuyerGuarantee/Routes/BuyerGuaranteeIndex.tsx

```diff
@@ -58,13 +60,29 @@ export const BuyerGuaranteeIndex: React.FC<BuyerGuaranteeIndexProps> = ({
 
   const learnMoreIcon = (
     <Flex pt={2} justifyContent="center">
-      <Text variant="mediumText" mt="-2px">
-        Learn More
-      </Text>
-      <ArrowRightIcon height="15px" width="15px" />
+      <Link
+        href="https://support.artsy.net/hc/en-us/articles/360048946973"
+        mt="-2px"
+        underlineBehavior="none"
+      >
+        <Flex>
+          <Text variant="mediumText">Learn More</Text>
+          <ArrowRightIcon height="15px" width="15px" mt="2px" ml="2px" />
+        </Flex>
+      </Link>
     </Flex>
   )
 
+  const navigateTo = "https://support.artsy.net/hc/en-us/articles/360048946973"
```

Comment:
Non-blocking: maybe something like `supportArticleURL` would give it more context.